lnbits/wallets/phoenixd.py

```diff
@@ -0,0 +1,169 @@
+import asyncio
+import base64
+import json
+import urllib.parse
+from typing import AsyncGenerator, Dict, Optional
+
+import httpx
+from loguru import logger
+from websockets.client import connect
+
+from lnbits.settings import settings
+
+from .base import (
+    InvoiceResponse,
+    PaymentPendingStatus,
+    PaymentResponse,
+    PaymentStatus,
+    StatusResponse,
+    Unsupported,
+    Wallet,
+)
+
+
+class PhoenixdWallet(Wallet):
+    """https://phoenix.acinq.co/server/api"""
+
+    def __init__(self):
+        if not settings.phoenixd_api_endpoint:
+            raise ValueError(
+                "cannot initialize PhoenixdWallet: missing phoenixd_api_endpoint"
+            )
+        if not settings.phoenixd_api_password:
+            raise ValueError(
+                "cannot initialize PhoenixdWallet: missing phoenixd_api_password"
+            )
+
+        self.endpoint = self.normalize_endpoint(settings.phoenixd_api_endpoint)
+
+        self.ws_url = f"ws://{urllib.parse.urlsplit(self.endpoint).netloc}/websocket"
```

Comment:
Why is the protocol `ws://` and not `wss://`?

Reply:
its like the cliche endpoint, phoenixd is also just running http. not https